.github/workflows/build_linux.yml

```diff
@@ -10,44 +10,49 @@ jobs:
         os: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04]
         build:
           - name: "Debug Build & Unit Tests (gcc)"
+            cmd_deps: sudo apt-get install -y -qq curl
```

Comment:
Lets install curl globally for all builds.
So we don’t Need to make the change in 20 different places….